### PR TITLE
fixed sample

### PR DESCRIPTION
**Pull request by jtrieu**
@JamesMBallard can you please review and merge?

### CODE REVIEW

File: ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/GraphicsOverlay_DictionaryRenderer/GraphicsOverlay_DictionaryRenderer.qml [hunk 1]
```diff
@@ -134,18 +121,17 @@ Rectangle {
...
                         }
                     }
                 }
 
                 // Zoom to graphics
-                if (bbox) {
-                    bbox = bbox.extent;
-                    mapView.setViewpointGeometryAndPadding(bbox, 20);
-                }
+                if(bbox)
```
Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/GraphicsOverlay_DictionaryRenderer/GraphicsOverlay_DictionaryRenderer.qml]: Let's keep the space between `if(` for consistency.

**Comment by JamesMBallard** [comment]
Review done. Please see comments.

File: ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/GraphicsOverlay_DictionaryRenderer/GraphicsOverlay_DictionaryRenderer.qml [hunk 2]
```diff
@@ -103,29 +101,18 @@ Rectangle {
...
-                            /**
-                             * If there are at least three points and the first and last points are
-                             * equivalent, assume it's a polygon.
-                             */
-                            builder = ArcGISRuntimeEnvironment.createObject("PolygonBuilder");
-                        } else {
-                            // Apparently it's a line
-                            builder = ArcGISRuntimeEnvironment.createObject("PolylineBuilder");
-                        }
+                        var builder = ArcGISRuntimeEnvironment.createObject("MultipointBuilder");
```
Comment (jtrieu) [inline ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/GraphicsOverlay_DictionaryRenderer/GraphicsOverlay_DictionaryRenderer.qml]: no. this is the workflow that we want to encourage our users to use

Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/GraphicsOverlay_DictionaryRenderer/GraphicsOverlay_DictionaryRenderer.qml]: Ok, sounds good.

**Comment by JamesMBallard** [comment]
:+1:

**Pull request merged**